src/khoj/routers/api_chat.py

```diff
@@ -58,6 +58,19 @@
 
 api_chat = APIRouter()
 
+from khoj.routers.email import feedback
+from pydantic import BaseModel
+
+
+class TextData(BaseModel):
+    uquery: str
+    kquery: str
+    sentiment: str
+
+@api_chat.post("/feedback")
+async def sendfeedback(data: TextData):
+    await feedback(data.uquery,data.kquery,data.sentiment)
```

Comment:
We should limit usage of this API only to authenticated users. And we should also include the email in case we need to follow-up.

```suggestion
@api_chat.post("/feedback")
@requires(["authenticated"])
async def sendfeedback(request: Request, data: TextData):
    user: KhojUser = request.user.object
    await feedback(data.uquery, data.kquery, data.sentiment, user.email)
```

Reply:
Got it.